scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -349,7 +354,7 @@ def get_coefficients(classifier, X_mat):
 ax = sns.distplot(predict_df.query("status == 1").decision_function, hist=False, label='Positives')
 
 
-# In[27]:
+# In[26]:
 
 ax = sns.distplot(predict_df.query("status == 0").probability, hist=False, label='Negatives')
 ax = sns.distplot(predict_df.query("status == 1").probability, hist=False, label='Positives')
```

Comment:
Can we add Y-axis labels, titles and maybe a brief comment or two to help explain these plots? It's not very easy to tell what they are showing, how they are different and how they could be used/interpreted.